modules/platforms/dotnet/Apache.Ignite/Internal/Table/Table.cs

```diff
@@ -36,7 +36,7 @@ internal sealed class Table : ITable
         private readonly ClientFailoverSocket _socket;
 
         /** Table id. */
-        private readonly IgniteUuid _id;
```

Comment:
@ptupitsyn Should we also remove IgniteUUID from ClientMessagePackType.cs?

Reply:
Good catch, fixed.